Subject: argtable2: remove malloc check from configure
From: Michael Olbrich <m.olbrich@pengutronix.de>

Without this patch "rpl_malloc" is used as malloc, which is wrong.

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>

---

Index: argtable2-7/configure.ac
===================================================================
--- argtable2-7.orig/configure.ac	2007-08-07 10:38:26.000000000 +0200
+++ argtable2-7/configure.ac	2007-08-07 10:38:33.000000000 +0200
@@ -26,7 +26,6 @@
 AC_STRUCT_TM
 
 # Checks for library functions.
-AC_FUNC_MALLOC
 AC_FUNC_STRFTIME
 AC_FUNC_STRTOD
 AC_CHECK_FUNCS([bzero strchr strcspn strrchr strtol])
